src/Api/Controllers/GroupsController.cs

```diff
@@ -133,7 +137,27 @@ public async Task Delete(string orgId, string id)
             throw new NotFoundException();
         }
 
-        await _groupService.DeleteAsync(group);
+        await _deleteGroupCommand.DeleteAsync(group);
+    }
+
+    [HttpDelete("")]
+    [HttpPost("delete")]
+    public async Task<ListResponseModel<GroupResponseModel>> BulkDelete([FromBody] GroupBulkRequestModel model)
+    {
+        var groups = await _groupRepository.GetManyByManyIds(model.Ids);
+
+        foreach (var group in groups)
+        {
+            if (!await _currentContext.ManageGroups(group.OrganizationId))
+            {
+                throw new NotFoundException();
+            }
+        }
+
+        await _deleteGroupCommand.DeleteManyAsync(groups);
+
+        return new ListResponseModel<GroupResponseModel>(groups.Select(g =>
+            new GroupResponseModel(g)));
```

Comment:
The front-end doesn't require all the details be returned, but I was trying to keep it consistent with the other admin service endpoints which return all the details during bulk deletion (`OrganizaitonUsersController` and `ProviderUsersController`). I didn't see any that just return `id`s, but I can do that if that's what we want to practice going forward.

I also see the `CiphersController` doesn't return anything for any of the delete many operations.

Reply:
The bulk *users operations return individualised success or error message for each user. (Try to bulk confirm users on your local instance and you'll see what I mean.) And we need to do that because we allow each operation on a user to succeed or fail separately, we don't abort a whole operation because some users couldn't be updated.

Let's only send information back to the client if we need it. So:
* if the UI allows for individualised error messages like this, or if there's a chance that only some groups will be affected but not all, then send a response back to the client
* if not, if it'll either succeed or fail and we can always know from the client what happened, then I think we just return an empty response (or 200 OK, whatever)

Reply:
Sounds reasonable to me! The delete endpoint/command is all or nothing, so there shouldn't be a need to return anything to the client besides a 200 OK.

Reply:
Done.